repos/2015-RH-IT-Root-CA.crt

```diff
@@ -0,0 +1,25 @@
+------BEGIN CERTIFICATE-----
```

Comment:
Maybe rename the `repos` dir to `pki` or `certs`?

Reply:
@cfergeau Done.